vuepress/docs/v7.0/docs/getting-started/concepts-overview.md

```diff
@@ -1,173 +1,167 @@
 # Entando Architecture Concepts
 
-A portal, website, web app, or mobile app built with Entando is called an Entando application.
-An Entando application is an assembly of out of the box and/or custom built components running on the
-Entando Platform. Components can be widgets, micro frontends, microservices, page templates, WCMS content or WCMS content types.
+Entando simplifies the process for enterprises looking to modernize across on-prem and cloud infrastructures with native support for Javascript app development, micro frontend and microservices architecture, deployment of apps in scalable containers, and automated management of containers with Kubernetes. 
+
+This document explores the architecture of the platform and some notable runtime characteristics.
+
+![entando-architecture](../entando-architecture.png)
```

Comment:
This looks like a screenshot of the full slide from the presentation with left and right background images. I think we had that before but can we clean that up and just include the diagram itself? We can probably drop the header as well but then when you click on the image you just get the interesting parts.